### ISSUE TITLE
Problem mixing HTTPS and insecure websockets

### ISSUE THREAD

**Issue by ToonTalk** [evothings/ecraft2learn#3]
If I run my Snap4Arduino code on an HTTPS page I get this error from Chrome:

mqttws.js:1046 Mixed Content: The page at 'https://snap.berkeley.edu/snapsource/snap.html' was loaded over HTTPS, but attempted to connect to the insecure WebSocket endpoint 'ws://raspberrypi.local:1884/mqtt'. This request has been blocked; this endpoint must be available over WSS.

**Comment by gokr** [comment]
Yeah, the fix is to get proper certs on the raspi and use wss instead of
plain ws.

Regards, Göran

Den 28 okt 2017 17:31 skrev "Ken Kahn" <notifications@github.com>:
